### PR TITLE
[PKG-3354] antlr4-python3-runtime v4.9.3

### PR DESCRIPTION
**Pull request by lorepirri**
[PKG-3354] antlr4-python3-runtime 4.9.3

- upstream: https://github.com/antlr/antlr4/tree/4.9.3
- dependency files:
    - https://github.com/antlr/antlr4/blob/4.9.3/runtime/Python3/setup.py 
- conda-forge: https://github.com/conda-forge/antlr-python-runtime-feedstock 
- pypi: https://pypi.org/project/antlr4-python3-runtime/ 

    
**Destination channel:** **defaults**

**Changes**
- exact version, new versioned feedstock
- fix recipe, remove entry point

**Dependency for**
- [omegaconf v2.3.0 ❄️](https://anaconda.atlassian.net/browse/PKG-3353)

[PKG-3354]: https://anaconda.atlassian.net/browse/PKG-3354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,14 +8,12 @@ package:
...
-  sha256: a53de701312f9bdacc5258a6872cd6c62b90d3a90ae25e494026f76267333b60
+  sha256: f224469b4168294902bb1efa80a8bf7855f24c99aef99cbefc1bcd3cce77881b
 
 build:
   number: 0
-  skip: true  # [py<30]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation  --ignore-installed --no-cache-dir -vvv
-  entry_points:
-    - pygrun=antlr4._pygrun:main
+  skip: true  # [py<37]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
Not needed

**Review by skupr-anaconda — APPROVED** [review]
LGTM but remove skipping

**Comment by M-Waszkiewicz-Anaconda** [comment]
we downgrade the package here, is not the case described here: 
https://github.com/anaconda-distribution/perseverance-skills/blob/ccd5ec3a8f2a26321a857d72c42f49eae4dfc90c/sections/02_Package_building/01_How_tos/Creating_versioned_feedstocks_from_existing_packages.md#creating-versioned-feedstocks-from-existing-packages

where we should create a separate git submodule ?

**Comment by lorepirri** [comment]
> we downgrade the package here, is not the case described here: https://github.com/anaconda-distribution/perseverance-skills/blob/ccd5ec3a8f2a26321a857d72c42f49eae4dfc90c/sections/02_Package_building/01_How_tos/Creating_versioned_feedstocks_from_existing_packages.md#creating-versioned-feedstocks-from-existing-packages
> 
> where we should create a separate git submodule ?

It should be exactly that case. I created the submodule in the folder `antlr4-python3-runtime-4.9.3-feedstock` (note the version), as described in the link.

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: ok

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Comment by lorepirri** [comment]
> antlr4-python3-runtime 4.9.3

@M-Waszkiewicz-Anaconda note also that I am merging into `master-4.9.3` and not `master`. The submodule will use `master-4.9.3` as base branch

**Pull request merged**